From fb7701e1f00fb9145f0f5b88f7d67fd3931f1e1b Mon Sep 17 00:00:00 2001
From: shinchiro <shinchiro@users.noreply.github.com>
Date: Sat, 6 Apr 2024 11:38:08 +0800
Subject: [PATCH] Do not use dirent.h

Because this causes some quirks on some partitions on Windows.
Relevant issue: https://github.com/mpv-player/mpv/issues/10679
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meson.build b/meson.build
index dd3cfe3..be3ea1e 100644
--- a/meson.build
+++ b/meson.build
@@ -84,7 +84,6 @@ python3 = import('python').find_installation()
 pytest = find_program('pytest', required: false)
 
 check_headers = [
-  ['dirent.h'],
   ['dlfcn.h'],
   ['fcntl.h'],
   ['inttypes.h'],
-- 
2.47.0

